tests/core/query/test_relational_query.py

```diff
@@ -132,6 +126,40 @@ def test_left_exclusive_and_right_join(sdata_query_aggregation):
     )
 
 
+def test_match_rows_join(sdata_query_aggregation):
```

Comment:
Very very good! I see the test for `match_rows='no'` is not included. Do you think it can be skipped or should it be also here?

Reply:
I think it can be skipped to be honest